homeassistant/components/unifiled/light.py

```diff
@@ -55,7 +39,8 @@ def __init__(self, light, api):
         self._api = api
         self._light = light
         self._name = light["name"]
-        self._unique_id = light["id"]
+        self._unifiledid = light["id"]
+        self._unique_id = "light." + light["id"]
```

Comment:
unique IDs don't need to be scoped by platform, Home Assistant already does that .

Reply:
Fixed this.